package.json

```diff
@@ -25,6 +25,7 @@
     "bundle-collapser": "^1.2.1",
     "chai": "^4.1.0",
     "chalk": "^2.0.0",
+    "charcodes": "0.1.0",
```

Comment:
Is this being used somewhere? I'd have thought the import in Babylon would be enough

Reply:
> Since we run flow from root, it couldn't resolve charcodes in babylon. I originally used --hoist with lerna, but @hzoo mentioned it was buggy, and --hoist charcodes wouldn't build (likely related to lerna/lerna#1028)... so for now, added it as a top level dep, not great, but couldn't get anything else to work that wasn't also just as hacky. Open to suggestions!

I tried a number of things to get it to resolve, but couldn't. I'll revisit it after digging more into why Travis is reporting zero errors.

Reply:
Sounds good. Flow is definitely smart enough to use the definitions relative to the module using the package, so that's very weird.